sunpy/io/ana.py

```diff
@@ -115,10 +108,12 @@ def write(filename, data, comments=False, compress=1, debug=False):
 
     Examples
     --------
-    >>> written = sunpy.io.ana.write(filename, data, comments=Falsem, compress=1)   # doctest: +SKIP
+    >>> written = sunpy.io.ana.write(filename, data, comments=False, compress=True)  # doctest: +SKIP
     """
     if _pyana is None:
-        raise ImportError("C extension for ANA is missing, please rebuild")# pragma: no cover
+        raise ImportError("C extension for ANA is missing, please rebuild")
+
+    compress *= 1
```

Comment:
This feels like a weird hack, I would say it either needs a comment, or a better way would be to just do `int(compress)` in the function call below.